src/transformers/integrations/integration_utils.py

```diff
@@ -960,9 +960,9 @@ def setup(self, args, state, model):
             remote server, e.g. s3 or GCS. If set to `True` or *1*, will copy each saved checkpoint on each save in
             [`TrainingArguments`]'s `output_dir` to the local or remote artifact storage. Using it without a remote
             storage will just copy the files to your artifact location.
-        - **MLFLOW_TRACKING_URI** (`str`, *optional*, defaults to `""`):
-            Whether to store runs at a specific path or remote server. Default to an empty string which will store runs
-            at `./mlruns` locally.
+        - **MLFLOW_TRACKING_URI** (`str`, *optional*, defaults to `None`):
+            Whether to store runs at a specific path or remote server. Default to `None` which skips setting the
+            tracking URI entirely.
```

Comment:
Another convention is the refer to the param being unset if defaults to `None` 

```suggestion
            Whether to store runs at a specific path or remote server. Unset by default, which skips setting the
            tracking URI entirely.
```